README.md

```diff
@@ -66,7 +66,7 @@ Documentation: https://mmdetection3d.readthedocs.io/
 
 English | [简体中文](README_zh-CN.md)
 
-The master branch works with **PyTorch 1.6+**.
+The main branch works with **PyTorch 1.8+**.
```

Comment:
Why not support PyTorch 1.6? Is there any reference?

Reply:
The update of MMCV: OpenMMLab 2.0 series only supports pytorch1.8+